src/ChunkMap.cpp

```diff
@@ -44,99 +44,35 @@ cChunkMap::cChunkMap(cWorld * a_World) :
 		)
 	)
 {
-
-}
-
-
-
-
-
-cChunkMap::~cChunkMap()
-{
-	cCSLock Lock(m_CSLayers);
-	while (!m_Layers.empty())
-	{
-		delete m_Layers.back();
-		m_Layers.pop_back();  // Must pop, because further chunk deletions query the chunkmap for entities and that would touch deleted data
```

Comment:
Can anyone verify that this removal is correct? How does C++ go about automatically cleaning up data structures?